genesis/utils/geom.py

```diff
@@ -1187,6 +1186,8 @@ def pos_lookat_up_to_T(pos, lookat, up, *, dtype=np.float32):
     z_norm = np.linalg.norm(z)
     if z_norm < gs.EPS:
         z = np.array([0.0, 1.0, 0.0], dtype=dtype)
+    if z.ndim == 2:
```

Comment:
In what scenarios whould z have ndim == 2? Why would lookup and pos have ndim == 2?